soak_bench_scripts/soak.sh

```diff
@@ -38,4 +39,10 @@ rm target/release/integration_tests-*.d
 printf "Starting soak tests...\n"
 
 # Loop the test forever until it hits an error
-loop -q target/release/integration_tests-* |& tee -a -i -p --output-error=warn $LOGFILE
+os=$(uname)
+echo $os |& tee $LOGFILE
+if [[ $os == "Darwin" ]]; then
+	loop -q target/release/integration_tests-* |& tee -a -i $LOGFILE
+else
+	loop -q target/release/integration_tests-* |& tee -a -i -p --output-error=warn $LOGFILE
+fi
```

Comment:
I think it's normal (at least in MaidSafe) to prefer spaces over tabs, and to have a newline at the end of files.  You can probably configure your editor to do this automatically for you.

Reply:
Noted and will update.

Reply:
Incidentally btw, this was the first time I ever seen `loop` used in Bash. Searching for `bash loop keyword` on Google doesn't seem to bring up any results for it.

Just out of curiosity, is there a reason to prefer this over a normal looping construct?

Reply:
It was the way Fraser had written is original script, I just went with it.

Reply:
@jacderida the `loop` used here is actually a tool that Fraser wrote a while back, see https://gitlab.com/Fraser999/loop
It's installed on the soak test machine.